cosmos/helpers.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import annotations
```

Comment:
Could we have a better name, other than `helpers`? Something along side `io`, `remote` or `storage`?

Reply:
chaned to io